terraform/main.tf

```diff
@@ -118,7 +118,7 @@ resource "google_dataflow_flex_template_job" "zkevm_imtbl_mainnet_job" {
     project = "${terraform.workspace}-im-data"
     runner = "DataflowRunner"
     workerMachineType = "n1-standard-1"
-    maxNumWorkers = 1
+    maxNumWorkers = 4
```

Comment:
Hey @jasadams-imtbl - Any specific insights or issues you came across that prompted this increase?

Reply:
not really @roopak-immutable, I just realised that with the maximum workers set to 1 we effectively have autoscaling disabled. With the transaction growth we were experiencing, it won't be much longer before we will need at least 2 workers. Just getting ahead of the game and creating some headroom. FYI, its still been sitting at 1 worker since I changed it in dev